src/apps/user/models.py

```diff
@@ -37,14 +41,23 @@ class Meta:
 
 
 class UserProfile(models.Model):
-    profile_picture = models.ImageField(upload_to="profile_pictures/", blank=True, null=True)
+    @staticmethod
+    def profile_picture_path(instance, filename):
+        # Generate a UUID and use it as part of the filename
+        ext = filename.split(".")[-1]
+        filename = f"{uuid.uuid4()}.{ext}"
+        return f"profile_pictures/{filename}"
+
+    profile_picture = models.ImageField(
+        upload_to=profile_picture_path,
+        storage=get_storage(StorageType.PRIVATE),
```

Comment:
Well, it doesn't work like that, it will trigger new migrations after we change MINIO setting